tinker/events/events_controller.py

```diff
@@ -77,15 +78,10 @@ def _iterate_child_xml(self, child, author):
 
         dates = child.find('system-data-structure').findall('event-dates')
         dates_str = []
-        unconverted_dates_list = {}
         for date in dates:
             try:
                 start = int(date.find('start-date').text) / 1000
                 end = int(date.find('end-date').text) / 1000
-                unconverted_dates_list.append({
```

Comment:
is this used anywhere else? Just wanted to make sure this is not being lost randomly